life_hc/MySQL/General/fn_cleanup.sqf

```diff
@@ -3,23 +3,28 @@
     File: fn_cleanup.sqf
     Author: Bryan "Tonic" Boardwine
 
-    This file is for Nanou's HeadlessClient.
-
     Description:
-    Server-side cleanup script on vehicles.
-    Sort of a lame way but whatever.
+    Server-side cleanup script on vehicles, dealers and fed reserve.
 */
 private _saveFuel = LIFE_SETTINGS(getNumber,"save_vehicle_fuel") isEqualTo 1;
 private _minUnitDistance = LIFE_SETTINGS(getNumber,"vehicles_despawn_max_distance");
 
-for "_i" from 0 to 1 step 0 do {
-    uiSleep (60 * 60);
+private _fnc_fedDealerUpdate = {
+    {
+        _x setVariable ["sellers",[],true];
```

Comment:
Perhaps it's time to save ourselves some time helping people on the Discord, and check if they're nil first

```suggestion
        private _dealer = missionNamespace getVariable [_x, nil];
        if (!isNil "_dealer") then {
            _dealer setVariable ["sellers", [], true];
        };
```